src/plugins/blocksmith/components/imgur-post-block-preview/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import ImgurPostBlockEmbed from './embed';
+
+class ImgurPostBlockPreview extends Component {
```

Comment:
@jalopez13 if you add a shouldComponentUpdate based on the block's etag we can avoid some of the needless renders from the heartbeat

also lets remove the script on unmount

Reply:
@efjacobson 

Leaving as key={Math.random()} so the preview can re-render when show_context is toggled.